lib/mappers/figma/gradient.js

```diff
@@ -0,0 +1,40 @@
+const formatColor = require('../sketch/color');
+const { tokenize, mapColorKeys } = require('../../utils');
+
+// Property locations differ with regular gradients and shadow gradients
+const getGradientPropertyLocation = gradient =>
+  gradient.gradientHandlePositions ? gradient : gradient.children[0].fills[0];
+
+const getGradientStops = gradient => {
+  const { gradientStops } = getGradientPropertyLocation(gradient);
+  return {
+    stops:
+      gradientStops !== undefined
+        ? gradientStops.map(stop => ({
+            position: stop.position,
+            color: formatColor(mapColorKeys(stop.color)),
+          }))
+        : null,
```

Comment:
i would expect stops to always be an array, so an empty one in this case. Verify how we do this in Sketch

Reply:
This was my bad, normally all gradients should have stops. In Sketch there is no case defined when there are no stops.